crypto/src/base58.rs

```diff
@@ -17,6 +17,8 @@ pub enum FromBase58CheckError {
     /// The input is missing checksum.
     #[fail(display = "missing checksum")]
     MissingChecksum,
+    #[fail(display = "mismatched data lenght")]
+    MismatchedLength,
```

Comment:
@akoptelov 
please, add here two attribtues:
- expected_length
- actual_length

it will really help to better investigate later error

Reply:
@akoptelov 
then rebase and merge to develop please